receiver/googlecloudmonitoringreceiver/receiver.go

```diff
@@ -314,8 +314,6 @@ func (mr *monitoringReceiver) convertGCPTimeSeriesToMetrics(metrics pmetric.Metr
 		mr.metricsBuilder.ConvertSumToMetrics(timeSeries, m)
 	case metric.MetricDescriptor_DELTA:
 		mr.metricsBuilder.ConvertDeltaToMetrics(timeSeries, m)
-	// TODO: Add support for HISTOGRAM
```

Comment:
I'm not sure why we are removing these.  We should keep them around until it is implemented.

Reply:
Yes, that's correct. 
I'll close the PR for now as it's important to implement this in the future.